boot.py

```diff
@@ -1,3 +1,13 @@
+import sublime
+from .plugin import __version__
+
+if __version__ <= (0, 11, 0) and int(sublime.version()) > 4000:
+    sublime.error_message(
```

Comment:
This entire check needs to move to plugin_loaded (also called startup in this plugin for some reason). Calling functions outside plugin_loaded is undefined.

Reply:
From documentation:
```
At importing time, plugins may not call any API functions, with the exception of sublime.version(), sublime.platform(), sublime.architecture() and sublime.channel().
```

Reply:
Hmm, I don’t see error_message among those functions, but I suppose that should always work too.

Reply:
Perhaps it's safer to raise an ImportError instead, looking at this example: https://github.com/SublimeText/AFileIcon/blob/master/plugin.py

Reply:
And not initialize LSP?
ST3 still works, just not as well, so I wouldn't block it completely.

As for `sublime.error_message`, I have overlooked that one. I also think it should just work as it shouldn't rely on anything in the API but we can ask maybe.